.github/workflows/checks.yml

```diff
@@ -22,6 +22,7 @@ jobs:
           ["3.8", "3.8-slim-buster"],
           ["3.9", "3.9-slim-buster"],
           ["3.10", "3.10-slim-buster"],
+          ["3.11", "3.11-slim-buster"]
```

Comment:
You say that you've successfully built Foca with py3.11 and py3.12. Any particular reason you're only including py3.11 here?

Reply:
Locally, there were no errors but `flake 8` check was not successful with version `3.12`. I am trying to solve it. Please give me some time.

Reply:
@uniqueg, Actually there is no python `3.12-slim-buster` image on Dockerhub as python 3.12 was released recently on 2nd October 2023. Although other images such as `3.12-slim-bookworm` and `3.12-slim-bullseye` are available, they have a lot of vulnerabilities and are in the development phase. One more image `python:3.12-rc-slim-buster` is available but it was published 3 months before python 3.12 release. What should I do?

Should we continue with just `3.11` for now?

Reply:
Good research.

Let's do the following: Add support for `3.11` only (keeping `3.8` for now), but use Debian `Bullseye` instead of `Buster` images (where available; not sure if the.

See [my new comment](https://github.com/elixir-cloud-aai/foca/issues/174#issuecomment-1863108270) in #174 for more details, as well as the new issue #199 (to be addressed in in mid-2024).

Reply:
Sure